Bio/PDB/Vector.py

```diff
@@ -69,7 +71,9 @@ def vector_to_axis(line, point):
 
 
 def rotaxis2m(theta, vector):
-    """Calculate a left multiplying rotation matrix that rotates
+    """Rotaxi2m method.
```

Comment:
How about ``Calculate left multiplying rotation matrix`` as the docstring one-line summary?